### DISCUSSION TITLE
Input mask / custom input format validation with Tailwind

### DISCUSSION THREAD

**Discussion by Alice1o4** [mdbootstrap/TW-Elements#1654] [Share ideas & request features]
I would like to be able to set a custom formatting for my inputs.

I.e. I will have an input field called "Main line code", but the main line codes have different formats in different counties. I will prepare a separate form for each county, but I want to make sure that the codes are inputted correctly.
I.e. one format is 3 letters, 2 numbers, 3 letters - xxx11xxx
And another one is 4 letters and 4 numbers - xxxx1111

I would like to set a condition on the input field that will only allow for inputting the codes in a proper format - preferably I could set a custom template via a data attribute on the input.

**Comment by filipkappa** [comment]
Good news! We introduced this feature in our latest release!
See here: #1899
